app.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import pickle
+
+import joblib
+from flasgger import Swagger
+from flask import Flask, request
+from flask_cors import CORS
+
+logging.basicConfig(level=logging.DEBUG, format="%(asctime)s %(levelname)s %(name)s : %(message)s")
+log = logging.getLogger(__name__)
+
+app = Flask(__name__)
+swagger = Swagger(app)
+CORS(app)
+
+
+@app.route("/", methods=["GET"])
+def home():
+    """
+    Default route
+    ---
+    consumes:
+      - application/json
+    responses:
+      200:
+        description: Some result
+    """
+    return {
+        "result": "Please use the /predict endpoint to predict the sentiment of a review text!",
+    }
+
+
+@app.route("/predict", methods=["POST"])
```

Comment:
Shouldn't this be a GET request?

Reply:
The review gets sent in the body of the request, which needs to be a POST in this case.